### PR TITLE
Bump kiwixlib version

### PR DESCRIPTION
**Pull request by mhutti1**
This fixes the issue with arm devices.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -36,7 +36,7 @@ dependencies {
 
   // Get kiwixlib online if it is not populated locally
   if (file("../kiwixlib/src/main").list().length == 1) {
-    compile 'org.kiwix.kiwixlib:kiwixlib:0.0.6'
+    compile 'org.kiwix.kiwixlib:kiwixlib:0.1.0'
```
Comment (mgautierfr) [inline app/build.gradle]: What is this version number ?

It is the official kiwix-lib release or it is yours ?
(kiwix-lib release is 0.2 and we will soon release a new version)

Comment (mhutti1) [reply app/build.gradle]: Its just an arbitrary number at the moment.

Comment (julianharty) [reply app/build.gradle]: I notice this PR is still open, despite the tests passing. I'm curious whether @mgautierfr would prefer the version number to match the native library. If not, can we merge this PR? :)

**Comment by mhutti1** [comment]
Fixes #240

**Comment by mhutti1** [comment]
I am going to merge this now as its imprortant. We do need to have a discussion about numbering though.

**Pull request merged**